bindings/core/src/method/account.rs

```diff
@@ -35,25 +35,26 @@ use serde::{Deserialize, Serialize};
 #[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(tag = "name", content = "data", rename_all = "camelCase")]
 pub enum AccountMethod {
-    /// Burn native tokens. This doesn't require the foundry output which minted them, but will not increase
+    /// Burn outputs.
+
+    /// When burn **native tokens**. This doesn't require the foundry output which minted them, but will not increase
     /// the foundries `melted_tokens` field, which makes it impossible to destroy the foundry output. Therefore it's
     /// recommended to use melting, if the foundry output is available.
-    /// Expected response: [`SentTransaction`](crate::Response::SentTransaction)
-    #[serde(rename_all = "camelCase")]
-    BurnNativeToken {
-        /// Native token id
-        token_id: TokenIdDto,
-        /// To be burned amount
-        burn_amount: U256Dto,
-        options: Option<TransactionOptionsDto>,
-    },
-    /// Burn an nft output. Outputs controlled by it will be swept before if they don't have a storage
+
+    /// When burn **nft** outputs. Outputs controlled by it will be sweeped before if they don't have a storage
     /// deposit return, timelock or expiration unlock condition. This should be preferred over burning, because after
     /// burning, the foundry can never be destroyed anymore.
+
+    /// When burn(destroy) an **alias** outputs. Outputs controlled by it will be sweeped before if they don't have a
+    /// storage deposit return, timelock or expiration unlock condition. The amount and possible native tokens will be
+    /// sent to the governor address.
+
+    /// When burn(destroy) **foundry** outputs with a circulating supply of 0.
+    /// Native tokens in the foundry (minted by other foundries) will be transactioned to the controlling alias.
```

Comment:
Actually there is no automatic sweeping, we did that at some point, but changed back because it required multiple transactions and wasn't really clear, so maybe you can remove it from the comments :sweat_smile:

Reply:
didn't notice, thanks, fixed